### DISCUSSION TITLE
Meeting Load issue

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1128] [Q&A] [answered]
<img width="1114" alt="Screen Shot 2023-05-15 at 11 46 55 am" src="https://github.com/code4recovery/spec/assets/23748349/39b324f1-3384-4b1f-be00-e38b50e2976a">
My meeting page was loading great a month ago and then my shortcode [tsml_react] stopped working. It would say there was a timezone issue? (screenshot attached)

_View page here:_ https://staging.al-anon.org.au/meetings/

There's also this error message

<img width="995" alt="Screen Shot 2023-05-15 at 11 54 37 am" src="https://github.com/code4recovery/spec/assets/23748349/bc468d06-4ac3-4d4f-8824-35f65c4279af">

I've got the meeting list back up but the layout is being weird. Do you know how I can fix this issue?

Thanks

**Comment by joshreisner** [comment, marked as answer]
hi @nataliefaith2 you can set your time zone in WordPress Dashboard > Settings. it looks like you solved it by switching to the legacy appearance, but the nice thing about the TSML UI appearance is that it has it shows upcoming meetings next. then you don't need to add the upcoming meetings widget to the meetings page itself

i see that maps are not loading with this error: `Uncaught Error: Use a public access token (pk.*) with Mapbox GL, not a secret access token (sk.*).` See https://www.mapbox.com/api-documentation/#access-tokens-and-token-scopes
